packages/pfff/pfff.0.37.1/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+maintainer: "yoann.padioleau@gmail.com"
+authors: [ "Yoann Padioleau <yoann.padioleau@gmail.com>" ]
+homepage: "https://github.com/facebook/pfff/wiki/Main"
+patches: [
+]
+build: [
+  ["./configure" "--novisual" "--nocmt" "--nobytecode"]
+  [make "depend"]
+  [make]
+  [make "opt"]
+]
+install: [
+  [make "install-libs"]
+]
+remove: [
+  ["./configure" "--novisual" "--nocmt" "--nobytecode"]
+  [make "uninstall-libs"]
+]
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "ocamlfind"
```

Comment:
```suggestion
```
Where is this used? I don't even see use of ocamlfind or Findlib anywhere in the source code

Reply:
There is some occurences of ocamlfind in some Makefile, but for opam related stuff it's not used indeed. I've removed the dependency. Thx.